code/game/objects/items/weapons/blades.dm

```diff
@@ -28,8 +28,9 @@
 	force = 60
 	attack_speed = 12
 	w_class = WEIGHT_CLASS_BULKY
-	flags_item = DRAINS_XENO
+	flags_item = DRAINS_XENO|TWOHANDED
```

Comment:
hello yes unrelated changes

Reply:
It needs the twohanded flag to be able to be wielded.